gnd/src/main/java/com/google/android/gnd/ui/editobservation/PhotoFieldViewModel.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.ui.editobservation;
+
+import android.net.Uri;
+import android.view.View;
+import androidx.databinding.ObservableMap;
+import androidx.databinding.ObservableMap.OnMapChangedCallback;
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import com.google.android.gnd.model.form.Field;
+import com.google.android.gnd.model.observation.Response;
+import com.google.android.gnd.system.StorageManager;
+import com.google.android.gnd.ui.common.AbstractViewModel;
+import javax.inject.Inject;
+
+public class PhotoFieldViewModel extends AbstractViewModel {
+
+  private final StorageManager storageManager;
+  private final MutableLiveData<Uri> destinationPath = new MutableLiveData<>();
```

Comment:
I'm a bit confused by the naming here - it appears Uris are loaded from destination paths, therefore they're not equivalent. If that's the case, shouldn't this be called `uri` instead?

Reply:
My bad, thanks for pointing it out

Reply:
Np! Thanks for all the careful code hygiene overall!